<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiNewDsSelector</class>
 <widget class="QDialog" name="MultiNewDsSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multiple Datasource Selection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Nr. Datasources</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="edgvComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>33</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>121</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QgsProjectionSelectionWidget" name="mQgsProjectionSelectionWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>30</horstretch>
       <verstretch>30</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>33</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>999999</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>321</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="okPushButton">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
